src/core/utils.js

```diff
@@ -115,16 +115,6 @@ export const nonNormativeSelector =
  *                    insertion point (Node) - or just appended, if
  *                    the element has no children.
  */
-export function makeOwnerSwapper(node) {
```

Comment:
Missed the comments for this function above 😅
Rest is all good 👌

Reply:
@sidvishnoi Yes, I did not realize those comments were for this function. Can you please check now?